### ISSUE TITLE
`referencePublication` and `usageCitation` recommendations and use cases are not well-defined

### ISSUE THREAD

**Issue by gremau** [EDIorg/dm-best-practices#92]
Seemingly anyways. Should we actually be recommending against using them? Review [here](https://prerelease-edi-docs.netlify.app/guide-eml-bp/chapter-08-citations-and-references).

**Comment by twhiteaker** [comment]
I don't think we should recommend **against** using them.  No need for an endorsement either, though.  The "cart before the horse" paragraph explains why it may be difficult to use them.  If you can do it, it's fine to do so. Otherwise, if you have alternate means of associating pubs with data, such as repository tools, then that's fine.  

There's this sentence, "If the resource is available with a DOIs at the time of EML creation, then they are fine to include, but alternative methods of linking these types of resources to a published EML dataset are typically preferred."  Preferred sounds like a recommendation to me, which I don't think we should be making here. There's also an extra "s" in "DOIs".  I suggest this:

`<referencePublication>` and `<usageCitation>` elements may be difficult to use in EML documents because the resource to be described is usually published after the EML document is created. If the resource is available with a DOI at the time of EML creation, then those elements are fine to include, but alternative methods of linking these types of resources to a published EML dataset may also be employed.

There was also a sentence saying referencePublication is a CitationType. It is actually a CitationListType.

I suggest removing example 8.3.  Sometimes people just skim examples and mimic them, and that example shows the not-recommended way. 

Since referencePublication and usageCitation are just CitationType elements, I think having the two existing citation element examples (8.1 and 8.2) is fine to show the structure.  However, the caption for the examples says this is for a reference publication, in which case referencePublication should be used instead of citation.  So, either change the example to be for something that would go as a child in literatureCited, or actually use **referencePublication**.
